pep-8100.rst

```diff
@@ -1,10 +1,9 @@
 PEP: 8100
 Title: January 2019 steering council election
-Version: $Revision$
-Last-Modified: $Date$
 Author: Nathaniel J. Smith <njs@pobox.com>, Ee Durbin <ee@python.org>
 Status: Active
```

Comment:
```suggestion
Status: Final
```

The election process has concluded and this SC is now historical, so this PEP should be marked accordingly